README.md

```diff
@@ -55,6 +55,7 @@ Select *Stash Pull Request Builder* then configure:
 - Rebuild if destination branch changes:
 - Build only if Stash reports no conflicts: this should be set if using the merge branch to avoid issues with out of data merge branch in stash
 - Build only if PR is mergeable (note this will stop the PR being built if you have required approvers limit set >0 and the PR hasn't been approved)
+- Probe the Stash REST API for merge status just so it recalculates the refspec: This probes the Stash REST API endpoint that causes recalculation of Git refspecs (see https://issues.jenkins-ci.org/browse/JENKINS-35219 and https://answers.atlassian.com/questions/239988/change-pull-request-refs-after-commit-instead-of-after-approval-or-workaround for details). Has no impact on proceeding with the build itself.
```

Comment:
tried to address this now

Reply:
Not so sure about clarifying the exact conditions, though... something was buggy, I investigated and fixed that, but got no certain science for that. :-)

Reply:
OK, please check the discussion here too https://github.com/jenkinsci/stash-pullrequest-builder-plugin/pull/22#issuecomment-460199716
Based on the research @proski  did and on the atlassian answer I'm suspecting there might be a problem with the build using stale commit, but it would *not* be fixed by triggering via the REST API

Reply:
I cannot really constructively comment on that... my guess would be that there are many different versions (and separate codebases) out there sharing the Bitbucket/Stash name and history, so behaviors probably vary. With the one(s) we have, there were problems using a wrong commit before this change (as posted originally), and none noted in the year and a half since then.

So my best guess is that at least for the deployment we have, it does solve the issue :-)

Reply:
That's good for you :), but a good description will also help other people.
Say somebody may have the same problem, but not realize it's a workaround for it.

If I guess your case correctly, maybe change `Has no impact on proceeding with the build itself.` to `Use this if you encounter problems with stale commits being built, but don't want to skip builds based on the PR status (like the two options above)`

Reply:
Also it might be useful to clarify this has no effect if either of the two options above are enabled.

Reply:
done, hopefully